src/badges.js

```diff
@@ -0,0 +1,163 @@
+import DrankAKegSmall from './resources/badges/70x70/DrankAKeg-70x70.png';
+import DrankFiveKegsSmall from './resources/badges/70x70/DrankFiveKegs-70x70.png';
+import DrankTenKegsSmall from './resources/badges/70x70/DrankTenKegs-70x70.png';
+import Edward40HandsSmall from './resources/badges/70x70/Edward40Hands-70x70.png';
+import FirstPourOfTheDaySmall from './resources/badges/70x70/FirstPourOfTheDay-70x70.png';
+import HatTrickSmall from './resources/badges/70x70/HatTrick-70x70.png';
+import KingOfTheKegSmall from './resources/badges/70x70/KingOfTheKeg-70x70.png';
+import LastPourOfTheKegSmall from './resources/badges/70x70/LastPourOfTheKeg-70x70.png';
+import LastPourOfTheNightSmall from './resources/badges/70x70/LastPourOfTheNight-70x70.png';
+import LightWeightSmall from './resources/badges/70x70/LightWeight-70x70.png';
+import PowerHourSmall from './resources/badges/70x70/PowerHour-70x70.png';
+import SevenDaysStraightSmall from './resources/badges/70x70/SevenDaysStraight-70x70.png';
+import WelcomeSmall from './resources/badges/70x70/Welcome-70x70.png';
+
+import BackOnTheBusLarge from './resources/badges/150x150/BackOnTheBus.png';
+import BeerAficionadoLarge from './resources/badges/150x150/BeerAficionado.png';
+import BeerAuthorityLarge from './resources/badges/150x150/BeerAuthority.png';
+import BeerBeforeNoonLarge from './resources/badges/150x150/BeerBeforeNoon.png';
+import BeerBuffLarge from './resources/badges/150x150/BeerBuff.png';
+import BeerConnoisseurLarge from './resources/badges/150x150/BeerConnoisseur.png';
+import DrankAKegLarge from './resources/badges/150x150/DrankAKeg.png';
+import DrankFiveKegsLarge from './resources/badges/150x150/DrankFiveKegs.png';
+import DrankTenKegsLarge from './resources/badges/150x150/DrankTenKegs.png';
+import Edward40HandsLarge from './resources/badges/150x150/Edward40Hands.png';
+import FirstPourOfTheDayLarge from './resources/badges/150x150/FirstPourOfTheDay.png';
+import HatTrickLarge from './resources/badges/150x150/HatTrick.png';
+import KingOfTheKegLarge from './resources/badges/150x150/KingOfTheKeg.png';
+import LastPourOfTheKegLarge from './resources/badges/150x150/LastPourOfTheKeg.png';
+import LastPourOfTheNightLarge from './resources/badges/150x150/LastPourOfTheNight.png';
+import LightWeightLarge from './resources/badges/150x150/LightWeight.png';
+import PowerHourLarge from './resources/badges/150x150/PowerHour.png';
+import SevenDaysStraightLarge from './resources/badges/150x150/SevenDaysStraight.png';
+import WelcomeLarge from './resources/badges/150x150/Welcome.png';
+
+export type Badge = {|
+  description: string,
+  image: { large: Object, small: Object },
+  name: string,
+|};
+
+export const BADGE_IMAGE_SIZES = {
+  large: 150,
+  small: 70,
+};
+
+// todo annotate badge type when implement achievement dao
+const BADGE_BY_TYPE: { [key: AchievementType]: Badge } = {
+  BackOnTheBus: {
+    description:
```

Comment:
I think I used to return these strings from the brewskey server but this is fine for now.

Reply:
yeah, in v1 api they come from the server on every profile request.